### PR TITLE
Fixes Puppet Forge wsgi application entry point

### PR DESCRIPTION
**Pull request by dkliban**
https://pulp.plan.io/issues/1599
closes #1599

### CODE REVIEW

File: pulp_puppet_plugins/pulp_puppet/forge/wsgi.py [hunk 1]
```diff
@@ -8,7 +8,39 @@
...
-application = get_wsgi_application()
+from pulp.server.webservices.application import SaveEnvironWSGIHandler
+from pulp.server import initialization, logs
+
+
+def wsgi_application():
+    """
+    Application factory to create, configure, and return a WSGI application
+    using the django framework
+    @return: wsgi application callable
```
Comment (bmbouter) [inline pulp_puppet_plugins/pulp_puppet/forge/wsgi.py]: I usually put a newline above the description and the return argument.

Comment (bmbouter) [inline pulp_puppet_plugins/pulp_puppet/forge/wsgi.py]: Also s/@return:/:return:/

**Comment by bmbouter** [comment]
@dkliban This makes sense to me. Can you verify that latest pulp-smash passes w/ this change also. Once you have, merge it!

**Comment by dkliban** [comment]
I ran pulp-smash on my dev environment and all the puppet tests passed.

**Pull request merged**